src/customJs/index.js

```diff
@@ -3,20 +3,23 @@ import { twitterPropertyConfig } from './socialTool/TwitterProperty';
 import { linkedInPropertyConfig } from './socialTool/LinkedInProperty';
 import { facebookPropertyConfig } from './socialTool/FacebookProperty';
 import { socialToolConfig } from './socialTool/SocialTool';
-import { socialShareToolConfig } from './socialShareTool';
-import { setCompanyTitle } from './sharedSingletonModule';
+import { getSocialShareToolConfig } from './socialShareTool';
+import { setCompanyTitle, setLocale } from './sharedSingletonModule';
 import { companyTitleDemoConfig } from './companyTitleDemo';
 
 const unlayer = window.unlayer;
 
-// Register Properties and tool Social Tool
-window.setCompanyTitle = setCompanyTitle;
-unlayer.registerPropertyEditor(twitterPropertyConfig);
-unlayer.registerPropertyEditor(linkedInPropertyConfig);
-unlayer.registerPropertyEditor(facebookPropertyConfig);
-unlayer.registerTool(socialToolConfig);
-unlayer.registerTool(companyTitleDemoConfig);
+window.initUnlayerExtensions = ({ locale, companyTitle }) => {
+  setCompanyTitle(companyTitle);
+  setLocale(locale);
+  unlayer.registerPropertyEditor(twitterPropertyConfig);
+  unlayer.registerPropertyEditor(linkedInPropertyConfig);
+  unlayer.registerPropertyEditor(facebookPropertyConfig);
+  unlayer.registerTool(socialToolConfig);
+  unlayer.registerTool(companyTitleDemoConfig);
+  // Register Properties and tool Social Share Tool
+  unlayer.registerTool(getSocialShareToolConfig());
```

Comment:
I think that we should refactor all config to be lazy (a function) in the same way that you did with `socialShareToolConfig`

```suggestion
  unlayer.registerPropertyEditor(getRwitterPropertyConfig());
  unlayer.registerPropertyEditor(getLinkedInPropertyConfig());
  unlayer.registerPropertyEditor(getFacebookPropertyConfig());
  unlayer.registerTool(getSocialToolConfig());
  unlayer.registerTool(getCompanyTitleDemoConfig());
  // Register Properties and tool Social Share Tool
  unlayer.registerTool(getSocialShareToolConfig());
```

Reply:
Resolved